utest/test/keywords/test_selenium_options_parser.py

```diff
@@ -114,6 +116,19 @@ def test_parse_options_string_errors(options, reporter):
     verify_all("Selenium options string errors", results, reporter=reporter)
 
 
+@pytest.mark.skipif(WINDOWS, reason="ApprovalTest do not support different line feeds")
```

Comment:
I think Approval tests did fix this problem and skipping in Windows could be removed now days. Doesn’t have to be this PR tough.